cookbook/docs/ml_training.rst

```diff
@@ -2,6 +2,8 @@
 ML Training
 ################
 
+Understand how Machine Learning models can be trained from within Flyte, along with orchestration benefits.
```

Comment:
```suggestion
Understand how machine learning models can be trained from within Flyte, with an added advantage of orchestration benefits.
```
"along with" doesn't sound right!

Reply:
added advantages and benefits convey the same message right? can we say something else? @samhita-alla